frontend/packages/data-portal/app/components/BrowseData/DatasetKeyPhoto.tsx

```diff
@@ -0,0 +1,39 @@
+import Skeleton from '@mui/material/Skeleton'
+
+import { useIsLoading } from 'app/hooks/useIsLoading'
+import { i18n } from 'app/i18n'
+import { cns } from 'app/utils/cns'
+
+export function DatasetKeyPhoto({
+  datasetTitle,
+  src,
+}: {
+  datasetTitle: string
+  src?: string
+}) {
+  const { isLoadingDebounced } = useIsLoading()
+  const containerClassName =
+    'min-w-[134px] min-h-[100px] max-w-[134px] max-h-[100px]'
```

Comment:
probably I just dont know tailwind well enough but why are we not just setting the width directly instead of including a min and max that are the same value?